src/js/components/DataTable/__tests__/__snapshots__/DataTable-test.tsx.snap

```diff
@@ -57364,8 +57364,8 @@ exports[`DataTable theme search pad, search text pad, sort gap, and expand size
   padding: 0;
   font-weight: inherit;
   text-align: inherit;
-  width: 48px;
-  max-width: 48px;
+  width: 768px;
+  max-width: 768px;
```

Comment:
Deploy preview looks good. What element are these snapshot changes related to? Seems like a big jump?